src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -507,7 +508,7 @@ bool emberAfOperationalCredentialsClusterUpdateFabricLabelCallback(app::CommandH
 exit:
     if (finalStatus == Status::Success)
     {
-        SendNOCResponse(commandObj, commandPath, OperationalCertStatus::kSuccess, ourFabricIndex, CharSpan());
+        SendNOCResponse(commandObj, commandPath, NodeOperationalCertStatusEnum::kOk, ourFabricIndex, CharSpan());
```

Comment:
Will this change to kOK in the future?

Reply:
@tcarmelveilleux That depends on what happens with https://github.com/project-chip/connectedhomeip/issues/24485